api/src/mongodb.ts

```diff
@@ -13,7 +15,10 @@ export default function (app: Application) {
     useNewUrlParser: true,
     useUnifiedTopology: true,
     serverSelectionTimeoutMS: serverSelectionTimeout*1000
-  }).then((client) => client.db(database))
+  }).then((client) => {
+    db_status = 200;
+    return client.db(database);
+  })
```

Comment:
Is there a scenario where we would need to switch from responding with a `200` to a `503` (e.g.: connection drops and cannot be re-established). Not sure whether a `catch` here would work or we would need to handle it elsewhere as well.

Reply:
It's possible, but the goal of this functionality is just to make sure that the api pod is able to form an initial connection in the openshift environment. ie: if the database pod hasn't started up yet, then openshift will keep restarting the api pod until a connection is formed.  

Previously whenever we issued a rollout to openshift the api pod would start up before the db pod and the connection would timeout from the api.

Reply:
If this is out of scope it could be added later if necessary. It just seemed like a good scenario to handle while we're in there.